.chloggen/1901-support-multi-cont-instr.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Is this a breaking change or enhancement?

Reply:
@pavolloffay when we switch the feature flag to Beta it will be listed as a breaking change.